_modules/index.html

```diff
@@ -47,11 +45,7 @@
     <ul class="nav nav-pills pull-left">
 	
         <li class="active"><a href="../index.html">Traits 5 User Manual</a></li>
-	
-  
-    <li><a href="#">Overview: module code</a></li>
```

Comment:
These changes are repeated throughout this PR. It's not clear to me what the cause is.

Reply:
This is a result of not using the `layout.html` in the `_templates` directory. I think the change is harmless. If we really want the extra button, we should put that into the theme.